pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -175,7 +178,7 @@ func NewMustGatherOptions(streams genericiooptions.IOStreams) *MustGatherOptions
 		SourceDir:        "/must-gather/",
 		IOStreams:        streams,
 		Timeout:          10 * time.Minute,
-		VolumePercentage: 70,
+		VolumePercentage: defaultVolumePercentage,
```

Comment:
Right above there is `/must-gather/` used as `SourceDir`, I guess we should use `defaultSourceDir` there?